build.gradle

```diff
@@ -117,22 +119,43 @@ pmd {
 }
 
 jacocoTestReport {
-  executionData(test, integration)
+  executionData(test)
   reports {
     xml.required = true
     csv.required = false
     xml.destination file("${buildDir}/reports/jacoco/test/jacocoTestReport.xml")
   }
 }
 
-project.tasks['sonarqube'].dependsOn jacocoTestReport
-project.tasks['check'].dependsOn integration
+pitest {
+  junit5PluginVersion = '0.15'
+  targetClasses = ['uk.gov.hmcts.reform.cdapi.*']
+  excludedClasses = [
+          'uk.gov.hmcts.reform.cdapi.config.*',
+          'uk.gov.hmcts.reform.cdapi.domain.*'
```

Comment:
These properties will need to be changed once the SDT commissioning code is added.  May also need to specify the testSourceSets property if the unit tests are not in the standard src/test directory (see https://github.com/hmcts/civil-sdt/pull/100 build.gradle).

Reply:
That's like a note rather than a directive to change something here

Reply:
Cool!